### PR TITLE
Implement slider

### PR DESCRIPTION
**Pull request by besnikkorca-ubt**
Added images for slider.
Do generic setup index, style.css

### CODE REVIEW

File: sliders/slide.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="slider.css">
+    <title>Document</title>
```
Comment (besnikkorca-ubt) [inline sliders/slide.html:8]: Update site title

**Review by BlendSejdiu — COMMENTED** [review]
Mire osht prof

File: sliders/slide.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                <img src="./Img/pht4.jpg" alt="image-4">
+            </div>
+        </div>
+
+        <!-- Controls -->
+        <div class="controls">
+            <button id="button-1" class="button">Image 1</button>
+            <button id="button-2" class="button">Image 2</button>
+            <button id="button-3" class="button">Image 3</button>
+            <button id="button-4" class="button">Image 4</button>
```
Comment (besnikkorca-ubt) [inline sliders/slide.html:34]: 2 buttons missing

**Review by besnikkorca-ubt — COMMENTED** [review]
Looks good

**Review by BlendSejdiu — APPROVED** [review]
test

**Pull request merged**